src/main/scala/cromwell/binding/values/WdlObject.scala

```diff
@@ -1,14 +1,41 @@
 package cromwell.binding.values
 
 import cromwell.binding.Call
-import cromwell.binding.types.{WdlCallOutputsObjectType, WdlObjectType}
+import cromwell.binding.types._
+import cromwell.util.{TryUtil, FileUtil}
+
+import scala.util.{Failure, Success, Try}
 
 trait WdlObjectLike {
   def value: Map[String, WdlValue]
 }
 
+object WdlObject {
+
+  def coerceObject(m: Map[String, String]): WdlObject = {
+    val coerced = WdlMap.coerceMap(m, WdlMapType(WdlStringType, WdlStringType)).value map {
+      case (k,v) => k.asInstanceOf[WdlString].value -> v
+    }
+    WdlObject(coerced)
+  }
+
+  def fromTsv(tsv: String): Try[WdlObject] = {
+    FileUtil.parseTsv(tsv) match {
+      case Success(table) if table.isEmpty => Success(WdlObject(Map.empty[String, WdlValue]))
```

Comment:
I'm not sure I see why it's ok to parse an empty tsv file to a map but not to an object though